leaf-ui/js/onFunctions.js

```diff
@@ -794,13 +798,19 @@ paper.on("link:options", function (cell) {
         for (let element of graph.getElements()) {
             var cell = element.findView(paper).model;
             var intention = cell.get('intention');
-            var initSatVal = intention.getUserEvaluationBBM(0).get('assignedEvidencePair');
-            var funcType = intention.get('evolvingFunction').get('type');
+            
+            if (intention != null) {
+                var initSatVal = intention.getUserEvaluationBBM(0).get('assignedEvidencePair');
+                if (intention.get('evolvingFunction') != null) {
+                    var funcType = intention.get('evolvingFunction').get('type');
```

Comment:
I still get an error if I "Clear Evaluation Labels" on a node that used to have an evolving function, but has been changed back to "No Function". (You have to click off the node to assign the evolving function, then click back onto the node to return it to "No Function".)

Reply:
<img width="318" alt="Screen Shot 2021-12-03 at 5 34 09 PM" src="https://user-images.githubusercontent.com/59470488/144681694-50f22493-3263-44bc-afe7-506016939962.png">